Finance/robinhood2.1m.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/local/bin/python3
+# -*- coding: utf-8 -*-
+
+# <bitbar.title>Robinhood2</bitbar.title>
+# <bitbar.version>v1.0</bitbar.version>
+# <bitbar.author>Adam Jilling</bitbar.author>
+# <bitbar.author.github>ajilling</bitbar.author.github>
+# <bitbar.desc>Shows your total portfolio value and daily change</bitbar.desc>
+# <bitbar.dependencies>python,robin_stocks</bitbar.dependencies>
```

Comment:
Hey - any chance you could add an image?

Reply:
Image added